server/src/domain/person/person.dto.ts

```diff
@@ -1,47 +1,37 @@
 import { AssetFaceEntity, PersonEntity } from '@app/infra/entities';
 import { ApiProperty } from '@nestjs/swagger';
 import { Transform, Type } from 'class-transformer';
-import {
-  IsArray,
-  IsBoolean,
-  IsDate,
-  IsNotEmpty,
-  IsOptional,
-  IsString,
-  ValidateIf,
-  ValidateNested,
-} from 'class-validator';
-import { toBoolean, ValidateUUID } from '../domain.util';
+import { IsArray, IsBoolean, IsDate, IsNotEmpty, IsString, ValidateNested } from 'class-validator';
+import { Optional, toBoolean, ValidateUUID } from '../domain.util';
 
 export class PersonUpdateDto {
   /**
    * Person name.
    */
-  @IsOptional()
+  @Optional()
   @IsString()
   name?: string;
 
   /**
    * Person date of birth.
    */
-  @IsOptional()
+  @Optional(true)
```

Comment:
I'd love the interface to be `({ nullable: true })` as it is easier to know what the option is doing.

Reply:
Yeah, it's unclear what `true` even means here